core/workflow-core/src/main/scala/edu/uci/ics/amber/util/IcebergUtil.scala

```diff
@@ -102,7 +102,7 @@ object IcebergUtil {
     catalog.initialize(
       catalogName,
       Map(
-        "warehouse" -> warehouse.toString,
+        "warehouse" -> warehouse.toString.replace(":", ""),
```

Comment:
Please add a comment here about why this is needed for Windows and why it also does not affect unix-based systems.